modern-font-stacks-for-wp.php

```diff
@@ -20,27 +20,21 @@
  *
  * @since 1.0
  * 
- * @param object $theme_json The original theme JSON data.
- * @return object The modified theme JSON data.
+ * @param WP_Theme_JSON_Data $theme_json The original theme JSON data.
+ * @return WP_Theme_JSON_Data The modified theme JSON data.
  */
 function mfswp_add_modern_font_stacks( $theme_json ) {
 	
 	// get current theme.json data
 	$current_data = $theme_json->get_data();
 
 	// bail if theme.json version is not version 2 (TODO: maybe show an admin notice that the plugin does not work?)
-	if ( empty( $current_data ) || ! isset( $current_data['version'] ) || ( $current_data['version'] !== 2 ) ) return $theme_json;
-
-	// get currently available font families
-	$current_font_families = $current_data['settings']['typography']['fontFamilies']['theme'];
-
-	// create the modern font stacks
-	$font_families = mfswp_get_modern_font_stacks();
-
-	// merge all font families
-	if ( is_array( $current_font_families ) ) {
-		$font_families = array_merge( $current_font_families, $font_families );
+	if ( ! isset( $current_data['version'] ) || $current_data['version'] !== 2 ) {
+		return $theme_json;
 	}
+
+	// merge current font families with the modern font stacks
+	$font_families = array_merge( $current_data['settings']['typography']['fontFamilies']['theme'], mfswp_get_modern_font_stacks() );
```

Comment:
Remove two temporary variables! 🎉

Reply:
# 💣

There is no `theme` key under `fontFamilies`.
https://github.com/WordPress/gutenberg/blob/77453a104d5c438ecf2aa21bc7bceef59b0f0d78/schemas/json/theme.json#L556